ledger-service/http-json/src/failurelib/scala/http/FailureTests.scala

```diff
@@ -46,9 +49,61 @@ abstract class FailureTests
   private def headersWithParties(actAs: List[domain.Party]) =
     Future successful headersWithPartyAuth(actAs, List(), Some(ledgerId().unwrap))
 
-  // TEST_EVIDENCE: Availability: Command submission succeeds after reconnect
-  "Command submission succeeds after reconnect" in withHttpService[Assertion] {
+  val availabilitySecurity: SecurityTest =
+    SecurityTest(property = Availability, asset = "Ledger Service HTTP JSON")
+
+  "Command submission succeeds after reconnect" taggedAs availabilitySecurity in withHttpService[
```

Comment:
Please make sure that the annotations include a scenario (either happy case  or attack). It is important information to understand what the test is actually about.

Reply:
This PR is mostly a mechanical transformation from TEST_EVIDENCE comments to the test-evidence scalatest tags.

As agreed in our call, I'll take care of this in another PR.